src/components/NavBar.jsx

```diff
@@ -0,0 +1,30 @@
+import { NavLink} from 'react-router-dom';
```

Comment:
- [ ] When I run the application. I got an error to give space just before `}`. Please fix that.